src/JSInterop/Microsoft.JSInterop.JS/src/package.json

```diff
@@ -25,8 +26,8 @@
     "dist/**"
   ],
   "devDependencies": {
-    "rimraf": "^2.5.4",
-    "tslint": "^5.9.1",
-    "typescript": "^2.7.1"
+    "rimraf": "^3.0.2",
+    "tslint": "^6.1.3",
```

Comment:
TSLint has been deprecated in favor of ESLint.

Reply:
Not changing linter; with different options, as part of this PR